sqlglot/expressions.py

```diff
@@ -28,7 +28,6 @@
     ensure_collection,
     ensure_list,
     seq_get,
-    split_num_words,
```

Comment:
delete this helper if it's not used

Reply:
I just used it in the bigquery method because we wanted to expand all needed values in the assignment, i.e.

```python
catalog, db, this, *rest = split_num_parts(...)
```

if we did a normal split it could fail to expand the values due to mismatch in the left hand side. For example in ``` `a.b` ``` it'd not find anything to bind to `this`.